src/main/charts/confluence/templates/_fluentd_templates.tpl

```diff
@@ -0,0 +1,37 @@
+{{- define "fluentd.container" -}}
```

Comment:
Rather than drop the YAML for the fluentd sidecar container and config volume into `statefulset.yaml`, I've defined them as templates in this new file, which are then included as fragments in the statefulset.